changes.d/pr-265.toml

```diff
@@ -0,0 +1,5 @@
+[[scuffle-http]]
```

Comment:
Can we add change logs for all the crates you update. Such as context and h3-webtransport.